src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java

```diff
@@ -464,6 +464,33 @@ public void testGivenAllelesMode() throws Exception {
         }
     }
 
+    /**
+     * Here we give Mutect2 ridiculous kmer settings in order to force assembly to fail.
```

Comment:
This was where the previous behavior was an infinite loop?

Reply:
Previous behavior was: assembly fails, therefore GGA alleles are injected in vain and don't get genotyped.  Assembly still fails now, but we then inject the GGA alleles into the reference without using a graph.